addr.go

```diff
@@ -34,6 +41,30 @@ func ParseAddr(s string) (*Addr, error) {
 	return &Addr{IPNet: m, Label: label}, nil
 }
 
+// ParseAddr parses the string representation of an address in the form $ip/$netmask.
+func ParseAddr(s string) (*Addr, error) {
+	m, err := ParseIPNet(s)
+	if err != nil {
+		return nil, err
+	}
+	return &Addr{IPNet: m}, nil
+}
```

Comment:
I'm still not following the logic here. Which AddrAdd takes a string? and why would you pass the same string to both of them?